backend/src/main/java/com/google/blackswan/mock/Constant.java

```diff
@@ -19,24 +19,41 @@
 
 /** Constants used throughout the blackswan mock classes. */
 public final class Constant {
+  // TODO: Create based on external config file (in the future).
   public static final String INTEREST_US = "Interest Over Time - US";
+  public static final String INTEREST_JP = "Interest Over Time - JP";
+  public static final String INTEREST_UK = "Interest Over Time - UK";
   public static final String UDON = "Udon";
   public static final String PHO = "Pho";
   public static final String RAMEN = "Ramen";
-  // TODO: Add more const for other metrics.
 
+  // TODO: Build with builder or for-loop.
   public static final ImmutableMap<DataInfo, String> FILE_LOCATIONS 
-      = ImmutableMap.of(
-          DataInfo.of(INTEREST_US, UDON), "udon-data.csv",
-          DataInfo.of(INTEREST_US, PHO), "pho-data.csv",
-          DataInfo.of(INTEREST_US, RAMEN), "interest-ramen.csv"
-        );
+      = ImmutableMap.<DataInfo, String>builder()
```

Comment:
This should be able to be built with some logic right since you have the general idea (ie dimension-country_code--data.csv) for the possible combinations.

And then have `Interest Over Time - ` as a separate constant and the different country codes as separate constant 'US', 'JP', 'UK", etc

Reply:
yep, i added a `TODO` for that initially, but now i implemented the logic